docs/examples/index.md

```diff
@@ -15,8 +16,29 @@ Check out these examples of different editor integrations. See the predefined bu
 </info-box>
 </span>
 
+## Predefined builds
+
+{@link installation/getting-started/predefined-builds Predefined CKEditor&nbsp;5 builds} are a set of ready-to-use rich text editors. Every “build” provides a single type of editor with a set of features and a default configuration. They provide convenient solutions that can be installed with no effort and that satisfy the most common editing use cases. See all of them in action: {@link examples/builds/classic-editor classic}, {@link examples/builds/inline-editor inline}, {@link examples/builds/balloon-editor balloon}, {@link examples/builds/balloon-block-editor balloon block}, {@link examples/builds/document-editor document}, and {@link examples/builds/multi-root-editor multi-root} editors! Learn their differences by using our examples and analyzing the source code of the builds.
+
+## Advanced custom builds
+
+CKEditor&nbsp;5 is a configurable framework that was created with collaboration in mind. It lets you build a custom editor of any type, with a wide set of features and the toolbar type that you need where multiple authors can easily work on the same rich text documents. Check out our {@link examples/builds/full-featured-editor feature-rich editor} that sports as many plugins as it is possible to put together. Want to invite some friends? Try the {@link examples/builds/collaborative-document-editor collaborative document editor} that brings the experience of the most popular online editing solutions to the table. And witness the flexibility of the UI in the {@link examples/custom/bottom-toolbar-editor toolbar-oriented example}.
```

Comment:
> CKEditor 5 is a configurable framework that was created with collaboration in mind. 

What about: "CKEditor 5 is a configurable framework created with collaboration in mind."

> Check out our feature-rich editor that sports as many plugins as it is possible to put together.

Sounds a little wordy to me. Maybe: "Check out our feature-rich editor that sports as many plugins as possible to put together."